src/lib.ts

```diff
@@ -108,8 +145,31 @@ export const args = Args.create("railo", "A script for farming elf stuff", {
     ],
     default: "both",
   }),
+  orb: Args.string({
+    options: [
+      ...[...Object.values(trainbots)]
+        .map((o) => Object.keys(o))
+        .flat()
+        .map((key) => [key, `Target the ${key} trainbot with the orb.`] as [string, string]),
+      ["none", "Don't use it!"],
+    ],
+    default: "none",
+  }),
 });
 
+let orbTarget: Monster | null = null;
```

Comment:
This might make things over-complicated so disregard if not interested: if we wanted to support the use case where someone wants to farm coal *OR* the familiar (e.g. more than one monster from the same train) we could make this instead `let orbTargets: Monster[] | null = null`

and then tweak the checks to run against all monsters in the list instead of just the one (in the prediction check and the validation)

Reply:
I'm going to declare this out of scope because I don't want to have to think about the Args for this, but this will be great for future evento scripts.

Reply:
If you want to accept a comma-separated list, you could consider `Args.strings` or even `Args.monsters` in grimoire